front/lib/resources/vault_resource.ts

```diff
@@ -288,9 +288,11 @@ export class VaultResource extends BaseResource<VaultModel> {
       case "system":
         return auth.isAdmin() && auth.canWrite([this.acl()]);
       case "global":
+      case "public":
```

Comment:
Let's not have the builder restriction here :+1:

Reply:
👌